Tests/OptimizelyTests-Common/BatchEventBuilderTests_Events.swift

```diff
@@ -76,6 +76,12 @@ class BatchEventBuilderTests_Events: XCTestCase {
         XCTAssertEqual(decision["campaign_id"] as! String, expCampaignId)
         XCTAssertEqual(decision["experiment_id"] as! String, expExperimentId)
         
+        let metaData = decision["metadata"] as! Dictionary<String, Any>
+        XCTAssertEqual(metaData["rule_type"] as! String, "experiment")
+        XCTAssertEqual(metaData["rule_key"] as! String, "ab_running_exp_audience_combo_exact_foo_or_true__and__42_or_4_2")
+        XCTAssertEqual(metaData["flag_key"] as! String, "")
```

Comment:
Does this test pass? I understand "flag_key" set to same as "rule_key" for activate.